src/include/job.h

```diff
@@ -600,14 +596,8 @@ struct job {
 	union jobextend {
 		char fill[256];	/* fill to keep same size */
 		struct {
-#if defined(__sgi)
-			jid_t	ji_jid;
-			ash_t	ji_ash;
-#else
 			char	ji_4jid[8];
```

Comment:
can we rename this to more meaning full name? Also please add comments for members which doesn't have